### PR TITLE
Return exit code from linux build script

### PR DESCRIPTION
**Pull request by digitalinfinity**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -151,4 +151,10 @@ echo Generating $BUILD_TYPE makefiles
 cmake $CMAKE_GEN $CC_PREFIX $ICU_PATH -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
 
 $MAKE $MULTICORE_BUILD 2>&1 | tee build.log
+_RET=${PIPESTATUS[0]}
+if [[ $_RET != 0 ]]; then
+    echo Error: $MAKE return $_RET
```
Comment (obastemur) [inline build.sh]: Can we put the message into quotes ? `"`

**Comment by obastemur** [comment]
LGTM

**Comment by obastemur** [comment]
@digitalinfinity this was the failing part on master right?

File: build.sh [hunk 2]
```diff
@@ -151,4 +151,10 @@ echo Generating $BUILD_TYPE makefiles
...
+fi
+
 popd > /dev/null
+exit $_RET
```
Comment (dilijev) [inline build.sh]: nit: newline at EOF

**Comment by dilijev** [comment]
LGTM

**Pull request merged**